doc/api/cli.md

```diff
@@ -595,6 +595,29 @@ Path to the file used to store the persistent REPL history. The default path is
 `~/.node_repl_history`, which is overridden by this variable. Setting the value
 to an empty string (`''` or `' '`) disables persistent REPL history.
 
+### `NODE_UNHANDLED_REJECTION=state`
```

Comment:
Maybe mention here the default behaviour?

Reply:
We do not do that anywhere else. However, it is described in the first paragraph, so I believe that should suffice.

Reply:
Let's not add a new environment variable. Let's just use the environment variable and allow it to be set in `NODE_OPTIONS`